MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>keep imports at the top</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>